frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -229,6 +260,43 @@ describe("QueryBuilder", () => {
 
         expect(screen.getByDisplayValue(TEST_CARD.name)).toBeInTheDocument();
       });
+
+      it("render time-series grouping widget for date field breakout", async () => {
+        await setup({
+          card: TEST_TIME_SERIES_WITH_DATE_BREAKOUT_CARD,
+        });
+        const timeSeriesModeFooter = await screen.findByTestId(
+          "timeseries-mode-bar",
+        );
```

Comment:
It's getting the footer which is this component
![Screenshot 2023-09-07 at 11 34 08 PM](https://github.com/metabase/metabase/assets/1937582/36cc4ce0-0e3a-4633-b5d8-a56c1103e1fc)